re_test.go

```diff
@@ -202,6 +202,52 @@ func TestReFunc(t *testing.T) {
 	}
 }
 
+func TestRePosition(t *testing.T) {
+	hp := `(\w+):(\d+)`
+	bytes := []byte("host:1234 host2:2345")
+	var pos re.Position
+	var host string
+	var port int
+	if err := re.Scan(regexp.MustCompile(hp), bytes, &pos, &host, &port); err != nil {
+		t.Fatalf("First match: unexpected error: %s", err)
+	} else {
```

Comment:
Drop the else and move the rest of the body out (Go style suggests not indenting after something like t.Fatal that has already returned from this function).